Library/Formula/testssl.rb

```diff
@@ -0,0 +1,19 @@
+class Testssl < Formula
+  desc "Tool which checks for the support of TLS/SSL ciphers and flaws"
+  homepage "https://testssl.sh/"
+  url "https://github.com/jsarenik/testssl.sh/releases/download/v2.6/testssl-2.6.tar.gz"
```

Comment:
This should be using the upstream release, not your own personal fork. Most of the audit errors are related to that.